containers/vue/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:14
+
+RUN npm install -g http-server @vue/cli @vue/cli-service-global
+WORKDIR /app
+COPY . .
```

Comment:
This will also copy all local source into the container which is not required for a dev container.  If you want to use the same Dockerfile for deployment and development, you can set up a multi-stage Dockerfile and use the `target` property. (Example is in https://github.com/microsoft/vscode-dev-containers/issues/304)

Reply:
I've removed the copy . . and the npm run serve from the dockerfile.